packages/jaspr/lib/src/ui/styles/properties/text.dart

```diff
@@ -26,7 +26,7 @@ class FontFamily {
   final String value;
 
   /// Constructs a [FontFamily] from a custom font name
-  const FontFamily(String value) : value = '"$value"';
```

Comment:
Font family names are supposed to be wrapped in double quotes: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family?retiredLocale=de

Reply:
The problem is it renders as a string within a string in inline styling, so I think it gets sanitized
![Screenshot 2024-01-18 at 9 25 50 AM](https://github.com/schultek/jaspr/assets/27149047/4fdd6828-e441-4ee9-93c0-f6c3c98e79be)

Reply:
I know the font name is wrong but it seems to work without the quotes and not with them

Reply:
https://stackoverflow.com/questions/32462939/inline-custom-css-fonts-in-html5

Reply:
Which browser are you using? For me it works also inline, event if its escaped.

Reply:
arc

Reply:
isn't that also chromium based?

Reply:
yup